packages/headless/src/features/analytics/generic-analytics-actions-loader.ts

```diff
@@ -44,6 +46,15 @@ export interface GenericAnalyticsActionCreators {
    * @returns A dispatchable action.
    */
   logCustomEvent(payload: LogCustomEventActionCreatorPayload): CustomAction;
+
+  /**
+   * Adds an entry in the action history.
+   * @param itemPermanentId - The unique identifier of the item.
+   * @returns A dispatchable action.
+   */
+  addPageViewEntryInActionsHistory(
+    itemPermanentId: string
+  ): AsyncThunkAction<void, string, {}>;
```

Comment:
How do you feel about adding that there @louis-bompart ? Is there another place which would make more sense?